src/app/pages/signing/disclaimer-button/disclaimer-button.component.html

```diff
@@ -0,0 +1,19 @@
+<a class="btn btn-danger disclaimer-button" data-toggle="popover" data-trigger="focus" tabindex="0"
```

Comment:
Popovers are generally for short tidbits of information, not large blocks like this. This popover looks awkward because it's too large. Suggested solutions:
- `ngIf` in the main sharing modal that displays this if an image tag is detected
- Use another modal
- Make the text shorter. The information can already found in the embed modal, right?

Reply:
I made it shorter in the newest commit. Check it out.